src/types/publication.types.ts

```diff
@@ -62,3 +50,66 @@ export interface DoiPublication {
   url: string;
   owner: string;
 }
+
+interface NpiDiscipline {
+  id: string;
+  title: string;
+}
+
+interface File {
+  title: string;
+  license: string; //enum?
+  acceptedVersion: boolean;
+  administrativeContract: boolean;
+  archivePublishDate: string; // date?
+}
+
+export interface Publication {
+  id: string;
+  modified: string; // date?
+  createdDate: string; // date?
+  createdBy: string;
+  title: { [key: string]: string };
```

Comment:
etterhvert bør det vel være en enum for key-type, slik at folk ikke kan skrive hva som helst. men kan ta dette når vi vet hvilke språk vi skal støtte